primitives/src/constants.rs

```diff
@@ -21,6 +21,7 @@ pub mod currency {
 	pub const FORTUNE: Balance = u128::MAX;
 	pub const UNITS: Balance = 1_000_000_000_000;
 	pub const DOLLARS: Balance = UNITS * 100; // 100 UNITS ~= 1 $
+	pub const GRAND: Balance = UNITS * 1_000;
```

Comment:
isn't a grand equal to a thousand of dollars? So it should be `DOLLARS * 1_000`?!

Reply:
yeah, especially looking at usage of it i think it should be DOLLARS * 1000

Reply:
I never understood the concept of DOLLARS
Should it be some approximation of $1 worth of BSX? Then it's wrong anyway (BSX price is 0.00009)

Reply:
originally it was approximation of the approximation of the price of BSX compared to HDX. if the limits should be in BSX, don't use confusing dollars nomenclature and just use UNITS

Reply:
but GRAND should be 1000 DOLLARS same as CENT being DOLARS / 100